MiniEngine/Core/GameCore.cpp

```diff
@@ -32,7 +32,11 @@ namespace GameCore
 
 	void InitializeApplication( IGameApp& game )
 	{
+#if (WINAPI_FAMILY == WINAPI_FAMILY_APP)
```

Comment:
According to winapifamily.h, WINAPI_FAMILY_APP is deprecated.  "Partitions" should be used to test for API support/app type.  Check the most inclusive API set first (desktop) using  WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP).